src/Http/ApiClient.php

```diff
@@ -0,0 +1,191 @@
+<?php
+namespace Jobsity\PhpTick\Http;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Exception\ServerException;
+use mef\Log\StandardLogger;
+
+/**
+ * Class ApiClient
+ *
+ * @package Jobsity\PhpTick\Http
+ */
+class ApiClient implements ClientInterface
+{
+    const BASE_URL = 'https://www.tickspot.com/';
+    const ENDPOINT_URL = '/api/v2/';
+
+    /**
+     * @var string User's subscription id
+     */
+    private $subscriptionId;
+
+    /**
+     * @var string User's access token
+     */
+    private $accessToken;
+
+    /**
+     * @var string User's company
+     */
+    private $company;
+
+    /**
+     * @var string User's email
+     */
+    private $email;
+
+    /**
+    * @var string API URL
+    */
+    private $apiUrl;
+
+    /**
+     * @var Client Guzzle Client Handler
+     */
+    private $client;
+
+    /**
+     * Constructs ApiClient
+     *
+     * @param string   $subscriptionId   Subscription id of the user.
+     * @param string   $accessToken      Access token of the user.
+     * @param string   $company          User's company.
+     * @param string   $email            User's email.
+     */
+    public function __construct($subscriptionId, $accessToken, $company, $email)
+    {
+        $this->subscriptionId = (string)$subscriptionId;
+        $this->accessToken = (string)$accessToken;
+        $this->company = (string)$company;
+        $this->email = (string)$email;
+
+        $this->apiUrl = self::BASE_URL . $this->subscriptionId . self::ENDPOINT_URL;
+
+        $this->client = new Client([
+            'headers' => [
+                'User-Agent' => $this->company . '(' . $this->email . ')',
+                'Authorization' => 'Token token=' . $this->accessToken
+            ]
+        ]);
+
+        $this->logger = new StandardLogger();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function get($endpoint, array $queryParams)
+    {
+        try {
+            $request = $this->client->request('GET', $this->apiUrl . $endpoint . '.json',
+                array('query' => $queryParams));
```

Comment:
Always use short array syntax:
```
['query' => $queryParams]
```
Also for multiline array look at constructor. 

Also if line does not pass the 120 char limit, then do not break it in multiple lines.